docs/calculations.mdx

```diff
@@ -437,42 +437,55 @@ creative_consumer_device_embodied_emissions_gco2pm =
 ### Media Delivery - Data Transfer
 
 ```
-session_seconds_per_imp =
-      property.average_seconds_per_session_ex_ads x
-      property.ad_funded_percentage /
-      property.average_imps_per_session
-
 media_data_transfer_emissions_gco2pm =
       media_data_transfer_usage_emissions_gco2pm +
       media_data_transfer_embodied_emissions_gco2pm
+```
+
+Determing the session seconds for a single impressions
 
 ```
+session_seconds_per_imp =
+      property.average_seconds_per_session_ex_ads x
+      property.ad_funded_percentage /
+      property.average_imps_per_session
+```
 
 For all channels other than CTV/BVOD, use the conventional model:
 
 ```
+media_data_transfer_kb_per_imp =
+      property.average_data_kb_per_session_seconds_ex_ads x
```

Comment:
Think we need to add in a default for when we don't have property specific values. Likely broken out by channel

Reply:
I think for CTV/audio this comes from the average bitrate, probably. For app-based channels let's pull an 80th percentile number based on what we have from data.ai... I think we could probably get web from a sampling of archive.org if we want, or can take 80th percentile of our database? This one would be helped with Sincera data prob.